### ISSUE TITLE
[BUG] Generated code produces code with error "Expected an identifier" in v2.13.1, was OK in v2.12.2.

### ISSUE THREAD

**Issue by RoarGronmo** [epam-cross-platform-lab/swagger-dart-code-generator#685]
**Describe the bug**
Got following error message on generated code: 
```
[SEVERE] json_serializable on lib/services/api/generated/m_office_bing_locations_query_api.swagger.dart:


line 412, column 62 of .: Expected an identifier.
    ╷
412 │ final  Map<String, dynamic>  $params = <String, dynamic>{'id$': id$, 'assetId': assetId, 'loggedTime': loggedTime, 'limit': limit, };
    │                                                              ^
    ╵

```

**To Reproduce**
Swagger used in this code generation:
[m_office_bing_locations_query_api.swagger.txt](https://github.com/epam-cross-platform-lab/swagger-dart-code-generator/files/13443972/m_office_bing_locations_query_api.swagger.txt)

**Expected behavior**
I want the code generation to produce flawless code as it did in v2.12.2.

**Swagger specification link**
Link to swagger/OpenApi documentation

**Library version used:**
v2.13.1

**Additional context**

**Comment by RoarGronmo** [comment]
Pinning version to v2.12.2 until fixed (or explained)!

**Comment by Vovanella95** [comment]
Hi @RoarGronmo , thanks for raising. Will take a look at it this week

**Comment by Vovanella95** [comment]
@RoarGronmo  fixed, published `2.13.2` :)

**Comment by RoarGronmo** [comment]
Sorry, but I checked again, and this time I got this error:

```
[SEVERE] json_serializable on lib/services/api/generated/m_office_bing_locations_query_api.swagger.dart:

UnimplementedError: (InvalidTypeImpl) InvalidType

```

It seems that it "skips" the generation or storing of the corresponding "g" file.

As you see in the picture below, the `m_office_bing_locations_query_api.swagger.dart` is missing its `m_office_bing_locations_query_api.swagger.g.dart` counterpart.

![image](https://github.com/epam-cross-platform-lab/swagger-dart-code-generator/assets/11217296/e6bc09d5-0bde-4d88-8260-be469fbf6c5d)


tried resetting flutter by `flutter clean` and resyncing afterwards, but it didn`t sort out...

reverting to v2.12.2 until resolved or explained.
